PRCS/prc-1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Paima Achievement Interface
+description: Interface for sharing in-game achievements.
+author: seba@paimastudios.com, edward@paimastudios.com
+status: open
+created: 2023-11-08
+---
+
+## Abstract
+
+A open cross-game standard achievement specification to gamify onchain participation. 
+This specification allows to interact through standard http methods.  
+
+## Motivation
+
+Most games have a achievement system, but there are not compatible: vendors and tools have adapt to each game. By implementing this standard, developer spend no or less time implementing the format itself, vendors can use the format for gamification apps, explorers, prizes, etc.
+
+## Format
+
+### HTTP
+* Network calls are done to any game node via HTTP to `BASE_URL`
+* All network requests are `Method GET`
+* All responses ContentType is `application/json`
+* `Standard HTTP codes` are used for status. E.g., 200 OK, 500 Internal Server Error, 404 Not Found, etc.  
+
+### Response Interfaces
+
+General Game Info
+```
+interface Game = {
+  id: string             // Game ID
+  name?: string          // Optional Game Name
+  version?: string       // Optional Game Version
+}
+```
+
+Data Validity
+```
+interface Validity = {
+  block: number;         // Data Block height (0 always valid)
+  chainId: number;       // Data ChainId
+  time: string;          // Optional Date ISO8601 YYYY-MM-DDTHH:mm:ss.sssZ
+}
+```
+
+Player Info
+``` 
+interface Player = {     
+  wallet:                // e.g., addr1234... or 0x1234..,
+  walletType?: 'cardano' | 'evm' | 'polkadot' | 'algorand' | string  // Optional wallet-type
+  userId?: string;       // If data for specific user: e.g., "1", "player-1", "unique-name", etc
```

Comment:
"if data"?

Reply:
Updated to mention that it must immutable for a specific account